### PR TITLE
45 hotfix light dark mode book appt page

### PR DESCRIPTION
**Pull request by DeeqaJamalini**
Darkmode styles added to bookappointment and added correct hex codes to dark mode colors in variables.scss

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -17,8 +17,9 @@ const App = () => {
   return (
     <Routes>
       <Route path="/" element={<Login />} />
-      <Route path="/home" element={<Home variant={theme} />} />
-      <Route path="/book-appointment" element={<BookAppointment />} />
+      <Route path="/home" element={<Home variant="light" />} />
```
Comment (fihockey) [inline src/App.tsx]: Please change the variant back to {theme}, this is how the toggle links to each element to change it from dark to light mode.

File: src/App.tsx [hunk 2]
```diff
@@ -17,8 +17,9 @@ const App = () => {
...
+      <Route path="/book-appointment" element={<BookAppointment variant="light"/>} />
```
Comment (fihockey) [inline src/App.tsx]: As well as here, the variant needs to be {theme}

File: src/App.tsx (hunk 1 above)
Comment (mh-fung) [inline src/App.tsx]: change it in theme

File: src/App.tsx [hunk 3]
```diff
@@ -17,8 +17,9 @@ const App = () => {
...
+      {/* <Route path="/book-appointment" element={<BookAppointment />} /> */}
```
Comment (mh-fung) [inline src/App.tsx]: Remove this comment

**Review by mh-fung — CHANGES_REQUESTED** [review]

File: src/pages/BookAppointment/BookAppointment.scss [hunk 4]
```diff
@@ -262,4 +315,93 @@
...
+  .book-appointment--dark {
+
+    .headerForm-container {
+      background-color: $clr-secondary-dk;
+      width: 100%;
+      height: 100vh;
+    }
+
+    .header {
+      box-shadow: none;
```
Comment (fihockey) [inline src/pages/BookAppointment/BookAppointment.scss]: There needs to be a shadow under the header on dark mode. Take a look at the Staff page and the settings page for the SCSS.

**Review by fihockey — CHANGES_REQUESTED** [review]

**Review by mh-fung — APPROVED** [review]

**Review by fihockey — APPROVED** [review]

**Pull request merged**